### DISCUSSION TITLE
Visual Studio Installer pkg

### DISCUSSION THREAD

**Discussion by benmccallum** [microsoft/winget-pkgs#4846] [Q&A]
It seems using the Microsoft.VisualStudioCommunity (or similar packages) are quite painful with regards to selecting your desired workloads and features. 

As such, what I'd like to do instead is be able to just install the Visual Studio Installer so I can launch that and then pick what I want (or import my .vsconfig) and setup that way.

Happy to create a manifest if someone can point me to a place where the Visual Studio Installer app could be downloaded from, but I have a feeling it might not be distributed standalone. So perhaps this needs to be a feature request to that team. 

Thanks!

**Comment by benmccallum** [comment]
Sort of relates to @RehanSaeed 's suggestion over on https://github.com/microsoft/winget-cli/issues/239 about a `--guided` option, but this wouldn't require anything in the CLI, just the ability to install "Visual Studio Installer" standalone.

**Comment by yigitemres** [comment]
> then pick what I want

Did you try `winget install microsoft.visualstudio.comunity --interactive`?

**Comment by denelon** [comment]
It's also possible to `winget install <package> --override <your parameters>`
